### PR TITLE
High DPI Human stuff

### PR DESCRIPTION
**Pull request by beccabunny**
High DPI of the human art upgrade

### CODE REVIEW

**Review by Amazinite — CHANGES_REQUESTED** [review]
Stream feedback:
* Maybe add some sort of "targeting detector" thing on the new gatling gun like the old one has.
* Meteor Missile Box isn't really a box anymore. I'm fine keeping it as is for the sake of simplicity, but it might be worth doing something along the lines of laying the rack in the new model flat and stacking more racks on top to make it look more box-like.
* Maybe make a new Brig outfit that actually shows the cell, similar to how the bunk rooms now have actual rooms to the image, just so that the Brig isn't the odd one out on "room" outfits. Might be fine to keep as is though given that the description says it's just the door that's added to an existing bunk room.
* The "spines" on the hyperdrive/scram drive seem a lot thinner. Maybe thicken them up a bit so they aren't so small.
* Maybe rotate the Syndicate shield generators a bit. Right now the camera is facing directly on one of the columns of generator bits.
* The old Stack Core broke up some detail by having those big brown pipes on the left side of the image. By removing those and adding an extra reactor part, the image is now rather monotonous. Maybe think about reintroducing those large pipes, or just doing something else to break up the image a bit.
* The fission and breeder reactors both have control rods on the top of the main body. The old fission reactor's control rods formed a cross, whereas the new fission reactor's form a circle. The breeder reactor, new and old, also has control rods that form a circle, so now both the fission and breeder reactors have circular control rod setups. Maybe go back to the cross on the fission reactor to give it a bit more distinction from the breeder reactor.
* The old fusion reactor had some extra bits around the main ring of the outfit, whereas the new fusion reactor is just the main ring. I feel like this causes the fusion reactor to be a bit too similar to the plasma cores. Maybe go back and add more detail around the ring as to break up the image a bit.
* The fusion reactor and Armageddon core are based off of modern attempts at creating fusion reactors. The fusion reactor is based off of the [tokamak design](https://en.wikipedia.org/wiki/Tokamak) and the Armageddon core is based off of the [laser confinement design](https://en.wikipedia.org/wiki/Laser_Inertial_Fusion_Energy). The new fusion reactor clearly shows this same design, being a torus structure, but the new Armageddon core design seems to push the whole "laser confinement" design aspect to the background. The old Armageddon core image largely consists of the structures that actually produce and focus the lasers, with the central sphere being a secondary characteristic of the image. Put differently, your eyes are drawn to the three structures that then focus down onto the central point. This new Armageddon core on the other hand first focus your eyes on the support mount on the top of the image, when then also puts your focus on the bottom mount. The fact that there is a top mount also causes the rest of the image to be shrouded in shadow, making it more difficult to make out what should be the important details of the reactor itself. I'd highly suggest going back to the drawing on this one. First remove the mounts, or at least the top mount of the image, as to now have them take away from what the image should be emphasizing. Then you should probably also make the structures that actually create and focus the lasers larger. On the old image they took up most of the image, but on this new one they are must less noticeable, being the last thing you notice after looking at the mounts and then the central sphere.
* The old fuel cells added a bit of color by having blue and green cylinders. You'd replaced some of these blue and greens with darker grays, though, making the fuel cells less colorful overall. Maybe bring some color back to them somehow.
* The old atomic engine design aesthetic is very striking. These new designs though feel rather bland by comparison to me (although they do look great). I know that the human and Hai atomic engines are rather similar in appearance already, so it isn't like replacing the human atomic engines would completely remove the design from the game, but I feel like between the human and Hai atomics, I'd rather keep the human atomics and replace the Hai atomics. Maybe change up the atomics you have here to fit a more "Hai" aesthetic? I could go either way on this, so I'd appreciate more feedback from others.
* The expansion images aren't "screwdrivers" anymore. Perhaps you should rename them to "cargo" and "outfits" expansions (and maybe we can leave the old screwdriver images in instead of removing them).
* How about readding the red fin on the back of the grab-strike?
image

**Review by Zitchas — CHANGES_REQUESTED** [review]
This restock of outfit images is a clear upgrade in the quality and detail of all the images provided, and a very nice improvement. 

My only concern is that most images seem to be both muted and darker. By this I mean that the colors are dimmer and less intense, and there is frequently more shadows. I don't know if the shadows are actually darker, but they certainly *feel* darker to me. It is as if the entirety of human outfitters all suddenly switched from ambient lighting to spotlights. Things definitely feel like everything is closer to being drab roughed-up-industrial sort of aesthetic than it used to be. The muting of colors isn't necessarily bad - it does look good, and it helps make alien stuff stand out a bit more. The shadows, on the other hand, I feel are a negative. I don't think we gain anything by having so much darkness.

Take the brig, for example. The main point of the outfit is the reinforced security doors, and yet in the image that whole wall and the doors are in shadow and difficult to make out. 

The gattling gun is another example of an image that has become much darker than it used to be; and this is intensified by having the body of the gattling gun switched to being a very dark grey material.

In terms of image design, however, I do have a concern:

The Large Rader Jammer: This outfit used to point outwards, which makes sense for an omni-directional effect. The new image has all three dishes pointing inwards, as if focussed on a single point located within the outfit. This makes me think of a microwave oven or some kind of scientific scanner for doing detailed scans of some small object located at the point of conjunction. I'd like to see them angled outwards again, or at least shifted so that their focus point is somewhere far outside the outfit instead of right in it. When you get right down to it, an omni-directional setup like the old radar jammer or either of the outfit scanners. The new radar jammer's focused direction would be a better look for the outfit scanner, which is a targeted effect.

My suggestion would be to swap bases: Have the Large Radar Jammer on the central pillar base of the outfit scanner, so all the dishes are pointed outwards; while the outfit scanner is put on the large radar jammer's base, and thus the dishes are focused on a single point. This would make their images convey the behaviour of the outfit.

In terms of reviews, this is basically a "98% approved, 2% request changes" sort of thing. If this all got merged as-is, I wouldn't be upset. But I do think fixing the Large Radar Jammer would be a definite good thing.  As for the shadows... Not sure what (if anything) can be done about them. Those aren't a deal breaker, just something I wanted to put out there for everyone to think about.

File: copyright [hunk 1]
```diff
@@ -353,11 +353,74 @@ Files:
...
+ images/outfit/small?radiothermal*
+ images/outfit/small?thermionic*
+ images/outfit/stack?core*
+ images/outfit/surveillance?pod*
+ images/outfit/banisher*
+ images/outfit/command?center*
+ images/outfit/fire-lance*
+ images/outfit/piercer*
+ images/outfit/piercer?launcher*
+ images/outfit/jorath?piercer?storage*
```
Comment (Amazinite) [inline copyright:418]:
```suggestion
 images/outfit/korath?piercer?storage*
```

File: copyright [hunk 2]
```diff
@@ -353,11 +353,74 @@ Files:
...
  images/outfit/rock?0*
  images/outfit/rock?1*
  images/outfit/rock?2*
  images/outfit/rock?3*
- images/outfit/rock?4*
+ images/outfit/roc
```
Comment (Amazinite) [inline copyright:423]: ?

**Pull request merged**